utils/node_local_test.py

```diff
@@ -200,7 +200,9 @@ def __del__(self):
         self.ts = None
         self.close()
 
-    def add_test_case(self, name, failure=None, test_class='core'):
+    def add_test_case(self, name, failure=None, test_class='core',
+                      output=None,
+                      duration=None):
```

Comment:
Not an issue with this PR, but should we at some point document function arguments in node_local_test.py? FWIW, we do try to do this with the functional test framework.
E.g.
```
    def add_test_case(self, name, failure=None, test_class='core',
                      output=None,
                      duration=None):
        """Add a test case to the results

        class and other metadata will be set automatically,
        if failure is set the test will fail with the message
        provided.  Saves the state to file after each update.

        Args:
            name (type): What this is.
            failure (type, optional): What this is.
            ...
        """
```

Reply:
We should, although the major difference is that ftest is more of a library and imported from elsewhere so you need to be conservative with changes, NLT less so.  This would sill be a nice to have.